src/management/apps.ts

```diff
@@ -224,3 +234,64 @@ function getListAppsResourceString(projectId: string, platform: AppPlatform): st
 
   return `/v1beta1/projects/${projectId}${resourceSuffix}`;
 }
+
+/**
+ * Gets the configuration artifact associated with the specified a Firebase app
+ * @return a promise that resolves to configuration file content of the requested app
+ */
+export async function getAppConfig(
+  appId: string,
+  platform: AppPlatform
+): Promise<AppConfigurationData> {
+  try {
+    const response = await api.request("GET", getAppConfigResourceString(appId, platform), {
+      auth: true,
+      origin: api.firebaseApiOrigin,
+      timeout: TIMEOUT_MILLIS,
+    });
+    return parseConfigFromResponse(response.body, platform);
+  } catch (err) {
+    logger.debug(err.message);
+    throw new FirebaseError(
+      `Failed to get ${platform} app configuration. See firebase-debug.log for more info.`,
+      {
+        exit: 2,
+        original: err,
+      }
+    );
+  }
+}
+
+function getAppConfigResourceString(appId: string, platform: AppPlatform): string {
+  let platformResource;
+  switch (platform) {
+    case AppPlatform.IOS:
+      platformResource = "iosApps";
+      break;
+    case AppPlatform.ANDROID:
+      platformResource = "androidApps";
+      break;
+    case AppPlatform.WEB:
+      platformResource = "webApps";
+      break;
+    default:
+      throw new FirebaseError("Unexpected app platform");
+  }
+
+  return `/v1beta1/projects/-/${platformResource}/${appId}/config`;
+}
+
+function parseConfigFromResponse(responseBody: any, platform: AppPlatform): AppConfigurationData {
+  if (platform === AppPlatform.WEB) {
+    return {
+      fileName: WEB_CONFIG_FILE_NAME,
+      fileContents: inspect(responseBody, { compact: false }),
+    };
+  } else if (platform === AppPlatform.ANDROID || platform === AppPlatform.IOS) {
+    return {
+      fileName: platform === AppPlatform.ANDROID ? ANDROID_CONFIG_FILE_NAME : IOS_CONFIG_FILE_NAME,
+      fileContents: Buffer.from(responseBody.configFileContents, "base64").toString("ascii"),
```

Comment:
The Admin SDK uses `utf8` instead of `ascii`, which I think is correct.

https://github.com/firebase/firebase-admin-node/blob/0547f4eaba5e791312eb55f128bdcb6bc3599501/src/project-management/android-app.ts#L127

If you change this, please also change the related comment in the `AppConfigurationData` interface.

Reply:
Done